crates/bevy_render/src/texture/image.rs

```diff
@@ -256,17 +256,34 @@ impl Image {
 
     /// Returns the aspect ratio (height/width) of a 2D image.
     pub fn aspect_2d(&self) -> f32 {
-        self.texture_descriptor.size.height as f32 / self.texture_descriptor.size.width as f32
+        self.height() as f32 / self.width() as f32
     }
 
-    /// Returns the size of a 2D image.
-    pub fn size(&self) -> Vec2 {
+    /// Returns the size of a 2D image as f32.
+    pub fn size_f32(&self) -> Vec2 {
         Vec2::new(
             self.texture_descriptor.size.width as f32,
             self.texture_descriptor.size.height as f32,
         )
     }
 
+    /// Returns the size of a 2D image.
+    pub fn size(&self) -> UVec2 {
```

Comment:
I think I'd prefer this one being named size_u32() and the f32 variant not changing. This way we don't introduce a breaking change for this.

Reply:
Yeah, I was thinking about that, but I renamed it because `size()` doesn't do any conversions, but `size_f32()` do conversion to f32 as the name describes it

Reply:
I prefer this naming: the non-converting form should have the better name.